api/src/test/resources/underlays/nautical.pbtext

```diff
@@ -362,6 +390,24 @@ entity_filters_schemas: {
     relationship_name: "sailor_reservation"
   }
 }
+hierarchies: {
+  attribute: {
```

Comment:
How will this work for OMOP? Would there just be one such hierarchy defined on the concept table? Or would there be different hierarchies depending on the specific domain table?

Reply:
So far, it's one hierarchy per entity.attribute. in this case `condition_occurrence.condition_concept_id`, where `procedures.procedure_concept_id` would have its own set of hierarchies. This let's hierarchical browsing be distinct to the domain.
Note that there are not multiple hierarchies per attributes - e.g. we would roll snomed and other condition vocabulary hierarchies together, they would just have different roots.